packages/gatsby/cache-dir/public-page-renderer-prod.js

```diff
@@ -1,9 +1,11 @@
 import React from "react"
 import PropTypes from "prop-types"
 
+import loader from "./loader"
 import InternalPageRenderer from "./page-renderer"
 
-const ProdPageRenderer = ({ location, pageResources }) => {
+const ProdPageRenderer = ({ location }) => {
```

Comment:
I wonder if we should keep that prop and fallback to `loader.loadPageSync` only if it's not passed?

Reply:
Well, the prop was added first in the change that broke it 😓 So there has never been a time when the prop was there and it worked, haha

https://github.com/gatsbyjs/gatsby/blame/3f11da3b68677a60de51025c388699392c66a594/packages/gatsby/cache-dir/public-page-renderer-prod.js#L6

I'd rather keep the public API as narrow (options wise) as possible.